pkg/operator/controllers/guardrails/gktemplates/aro-deny-delete.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: aro-deny-delete
```

Comment:
the name here should be all lower case of spec.crd.spec.names.kind, ie, arodenydelete, as required by gatekeeper